README.md

```diff
@@ -1,27 +1,33 @@
-# The Rust Programming Language
-
-[![Rust Community](https://img.shields.io/badge/Rust_Community%20-Join_us-brightgreen?style=plastic&logo=rust)](https://www.rust-lang.org/community)
+<p align="center">
+  <img alt="The Rust Programming Language: A language empowering everyone to build reliable and efficient software" src="https://www.rust-lang.org/static/images/rust-social-wide.jpg" width="512" height="256">
+  <br>
+  <a href="https://www.rust-lang.org/">Website</a> |
+  <a href="https://www.rust-lang.org/learn/get-started">Getting started</a> |
+  <a href="https://www.rust-lang.org/learn">Learn</a> |
+  <a href="https://www.rust-lang.org/learn#learn-use">Documentation</a> |
+  <a href="CONTRIBUTING.md">Contributing</a>
+</p>
 
 This is the main source code repository for [Rust]. It contains the compiler,
 standard library, and documentation.
 
 [Rust]: https://www.rust-lang.org/
 
-**Note: this README is for _users_ rather than _contributors_.**
-If you wish to _contribute_ to the compiler, you should read
-[CONTRIBUTING.md](CONTRIBUTING.md) instead.
+## Why Rust?
+
+### Performance
+
+Rust is blazingly fast and memory-efficient: with no runtime or garbage collector, it can power performance-critical services, run on embedded devices, and easily integrate with other languages.
```

Comment:
If Rust doesn't have a runtime, how do you explain the runtime startup?

https://github.com/rust-lang/rust/blob/master/library/rtstartup/rsbegin.rs

We both know what people mean, but this is the surface of the documentation for the language, it needn't be written in a marketing-speak tone.

Reply:
There is no longer any mention of "runtime".